src/Resources/Resource.php

```diff
@@ -424,6 +425,27 @@ public function withoutChannel()
     }
     #endregion
 
+    /**
+     * @param int|null    $userId
+     * @param string|null $fingerprint
+     *
+     * @return array|int|Entity|Collection
+     * @throws GuzzleException
+     */
+    public function getByUser(int $userId = null, string $fingerprint = null)
```

Comment:
As far as I see it is in base class. Why?

Reply:
Using as minimum in two child classes and will be used everywhere, where we will have functionality for getting user by id or fingerprint